content/docs/kubernetes/access-fluvio.md

```diff
@@ -1,11 +1,12 @@
 ---
-title: Network Endpoints
+title: Access Fluvio from Outside
+menu: Access Fluvio
 weight: 40
 ---
 
 ## Exposed Services
 
-By default Fluvio services are exposed via NodePort configuration.
+By default Fluvio services are exposed via NodePorts.
```

Comment:
NodePorts by default hasn't landed in `master` yet. We should say something like it is using Minikube's LoadBalancer. We'll just have to update this section after it gets merged in